MSGoal/LeetCode100/src/LongestSubstringWithoutRepCharac.java

```diff
@@ -0,0 +1,28 @@
+import java.util.*;
+
+ public  class LongestSubstringWithoutRepCharac {
+		
```

Comment:
Also make sure to add the difficulty you faced in this problm.. so its easy when you have to revise